README.md

```diff
@@ -89,9 +85,9 @@ We can characterize this code objectively and dispassionately, too:
 2. always sends an email, provided `send_email` "works"
 3. does nothing else
 
-**Our last iteration is objectively less complex.**
+**Our latest iteration is objectively less complex.**
 
-OK, well how can we think about complexity in general?
+So how can we think about complexity in general?
```

Comment:
Remove 'So` and just start with 'How can we think...'. I do the 'So' prefix unnecessarily too many times personally.